db/schema.rb

```diff
@@ -10,9 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_05_26_212613) do
+ActiveRecord::Schema.define(version: 2020_08_06_004459) do
```

Comment:
hey i see you removed the conversion to active storage migration.

this right now is not working on postgres. could you keep the migrations the same and try rewriting the docker stuff to work with mysql or sqlite. i know the migrations work there.

Reply:
I think I ran a migration and then rolled it back just trying to run the app to see how it works.  I wen back into the git branch for this pull request and it show 'ActiveRecord::Schema.define(version: 2020_08_06_004459)'.  Do you want images for both production and development for all three databases?

Reply:
it's just for demo. just mysql should be sufficient.